test/e2e/page-objects/home.page.js

```diff
@@ -0,0 +1,26 @@
+const BasePage = require('./base.page');
+
+class HomePage extends BasePage {
+  constructor(driver) {
+    // Selectors
+    super(driver);
+    this.btnSend = '[data-testid="eth-overview-send"]';
+    this.btnActivity = '[data-testid="home__activity-tab"]';
+    this.confirmedTx = '.transaction-status-label--confirmed';
+  }
+
+  // Methods
+  async startSendFlow() {
+    await this.driver.clickElement(this.btnSend);
+  }
+
+  async goToActivityList() {
+    await this.driver.clickElement(this.btnActivity);
+  }
+
+  async isConfirmedTxInActivity() {
+    return await this.driver.isElementPresent(this.confirmedTx);
+  }
```

Comment:
notice here we don't do any assertion, we only return a boolean. As stated in the guidelines, assertions should go on the spec level, not on the page object classes. See below how this method is called in the spec for doing the assertion there

Reply:
I propose we consider incorporating assertion methods within our page object. This approach could offer several advantages:
Code Reusability: By creating methods like check_confirmedTxDisplayedInActivity() and check_confirmedTxNotDisplayedInActivity(), we can reuse these assertions throughout our tests, saving coding time.
```
async check_confirmedTxDisplayedInActivity() {
  assert.equal(await isConfirmedTxInActivity(), true, 'Confirmed tx is not found');
}
async check_confirmedTxNotDisplayedInActivity() {
assert.equal(await isConfirmedTxInActivity(), false, 'Confirmed tx is displayed');
}
```
For the class organization:  We already have elements and action methods, when we add a section for check methods in the page class, we could consider adopting a naming convention such as check_xxx for assertions and action_xxx for actions. Something like action_ startSendFlow(), This approach would make it easier to locate specific methods within the class, especially when dealing with a large list of class methods
We can also add parameter for check methods, which allowing us to adapt the error message dynamically. For example:
```
async check_confirmedTxNumber(expectedNumberOfTx) {
  let currentNumberOfTx = txElements.length;
  assert.equal(
    expectedNumberOfTx, 
    currentNumberOfTx, 
    `We have ${expectedNumberOfTx} transactions appear instead of ${currentNumberOfTx}`
  );
}
```
This approach would automatically generate a helpful error message for debugging purposes. In fact, I would recommend that all check methods have their error messages defined within the class. Currently, in our tests, error messages are not always specified in the test spec. By defining error messages within the class, we can ensure that meaningful error messages are consistently used throughout our tests, making it easier for people to create and debug tests.